tower/src/hedge/mod.rs

```diff
@@ -4,11 +4,12 @@
 #![warn(missing_debug_implementations, missing_docs, unreachable_pub)]
 
 use crate::filter::AsyncFilter;
-use futures_util::future;
+use futures_util::future::Either;
```

Comment:
The tokio-util crate also has an `Either` type, so if `Either` is the only thing left we use from futures-util, we can get rid of the dependency. Not sure if that's the cause, though.

Reply:
The `hedge` feature depends on the `filter` and `util` features, which both pull in `futures-util` (and for the latter, that's a public dependency). So I don't think this is possible short-term.

Reply:
Alright, in that case it makes sense to keep it. :+1: